### ISSUE TITLE
UNAVCO test across repo's

### ISSUE THREAD

**Issue by dbekaert** [aria-tools/ARIA-tools#81]
Let us test with both releases for Mintpy and ARIA-tools. ARIA-tools is being released from the v1 branch not from dev, which contains additional functionality. We back-ported some of the changes wrt meta-data so should be ok. But might be good to do some tests @sssangha @ehavazli @hfattahi @yunjunz.

**Comment by yunjunz** [comment]
The latest released version of aria-tools+mintpy workflow works for me with the SanFran dataset.

**Comment by ehavazli** [comment]
I will be looking in to unwrapping error corrections I had and some GPS features this week but overall the process can be completed in my case.

**Comment by dbekaert** [comment]
ok great, I will lease this open for a bit longer. If there are issues that needs to be fixed, lets track it here as well, so we can back-port from dev to v1.

**Currently open for back-porting:**
- bugfix for mask and dem being a pixel off:  https://github.com/aria-tools/ARIA-tools/pull/83#issue-302373418

**Comment by dbekaert** [comment]
notebooks tested and good to go.
